client/my-sites/domains/components/mapping-instructions/index.jsx

```diff
@@ -30,41 +30,191 @@ class DomainMappingInstructions extends React.Component {
 			{
 				components: {
-					link: <ExternalLink href={ MAP_EXISTING_DOMAIN_UPDATE_A_RECORDS } target="_blank" />,
+					strong: <strong />,
+					link: <ExternalLink href={ MAP_SUBDOMAIN_WITH_CNAME_RECORDS } target="_blank" />,
 				},
 			}
 		);
+
+		const additionalInstructions = [
+			translate(
+				'Some DNS managers will only require you to add the subdomain (i.e. "%(subdomainPart)s") in a field typically labeled "host", "name" or "@".',
+				{
+					args: { subdomainPart },
+				}
+			),
+			translate(
+				'Some DNS managers will have the canonical name part (i.e. "%(canonicalName)s") labeled as "points to" or "alias".',
+				{
+					args: { canonicalName: wpcomDomainName },
+				}
+			),
+			translate( 'Remember to add the periods after the domain names.' ),
+			translate(
+				"If you're not sure how to do this, we recommend contacting your domain registrar for assistance."
+			),
+		];
+
 		return (
-			<FoldableFAQ id="advanced-mapping-setup" question={ advancedSetupUsingARecordsTitle }>
+			<React.Fragment>
+				{ isAtomic && (
+					<Notice status="is-warning" showDismiss={ false } translate={ this.props.translate }>
+						{ cnameMappingWarning }
+					</Notice>
+				) }
+
+				<p>{ cnameInstructionsMessage }</p>
+				<p>
+					<code>
+						{ domainName }. IN CNAME { wpcomDomainName }.
+					</code>
+				</p>
+				<ul>
+					{ additionalInstructions.map( ( instruction, i ) => (
+						<li key={ i }>{ instruction }</li>
+					) ) }
+				</ul>
+			</React.Fragment>
+		);
+	}
+
+	renderCnameInstructionsWithHeader() {
+		return (
+			<FoldableFAQ
+				id="alternative-mapping-setup-cname-record"
+				key="alternative-mapping-setup-cname-record"
+				question={ this.props.translate( 'Alternative setup using CNAME records' ) }
+			>
+				{ this.renderCnameInstructions() }
+			</FoldableFAQ>
+		);
+	}
+
+	renderARecordsList() {
+		const { aRecordsRequiredForMapping, domainName } = this.props;
+		return (
+			<ul className="mapping-instructions__dns-records-list">
+				{ aRecordsRequiredForMapping.map( ( aRecord ) => {
+					return (
+						<li key={ aRecord }>
+							<code>
+								{ domainName }. IN A { aRecord }
+							</code>
+						</li>
+					);
+				} ) }
+			</ul>
+		);
+	}
+
+	renderARecordsInstructionsWithHeader() {
+		const { domainName, translate } = this.props;
+
+		const advancedSetupUsingARecordsTitle = isSubdomain( domainName )
+			? translate( 'Alternative setup using root A records' )
+			: translate( 'Alternative setup using A records' );
+		const aRecordMappingWarning = isSubdomain( domainName )
+			? translate(
+					'If you map a subdomain using A records rather than WordPress.com name servers, you will need to manage your subdomain’s DNS records yourself for any other services you are using with your subdomain, including email forwarding or email hosting (i.e. with Google Workspace or Titan)'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'If you map a domain using A records rather than WordPress.com name servers, you will need to manage your domain’s DNS records yourself for any other services you are using with your domain, including email forwarding or email hosting (i.e. with Google Workspace or Titan)' )` *ES Score: 8*